ape_trezor/client.py

```diff
@@ -0,0 +1,114 @@
+from typing import Any, Dict, Optional, Tuple
+
+from eth_typing.evm import ChecksumAddress
+from trezorlib import ethereum  # type: ignore
+from trezorlib.client import get_default_client  # type: ignore
+from trezorlib.exceptions import PinException, TrezorFailure
```

Comment:
```suggestion
from trezorlib.exceptions import PinException, TrezorFailure  # type: ignore
```

^ this should finally make `mypy` happy